src/hadoop-data-node/deploy/hadoop-data-node-configuration/datanode-generate-script.sh

```diff
@@ -23,7 +23,5 @@ cp  /hadoop-configuration/hadoop-env.sh $HADOOP_CONF_DIR/hadoop-env.sh
 cp  /hadoop-configuration/yarn-env.sh $HADOOP_CONF_DIR/yarn-env.sh
 cp  /hadoop-configuration/mapred-site.xml $HADOOP_CONF_DIR/mapred-site.xml
 
-HOST_NAME=`hostname`
-/usr/local/host-configure.py -c /host-configuration/host-configuration.yaml -f $HADOOP_CONF_DIR/hdfs-site.xml -n $HOST_NAME
-
-sed -i "s/{HDFS_ADDRESS}/${HDFS_ADDRESS}/g" $HADOOP_CONF_DIR/core-site.xml 
+sed -i "s/{POD_IP}/${POD_IP}/g" $HADOOP_CONF_DIR/hdfs-site.xml
+sed -i "s/{HDFS_ADDRESS}/${HDFS_ADDRESS}/g" $HADOOP_CONF_DIR/core-site.xml
```

Comment:
Confused about your goal. If you wanna decouple cluster-object-model from the service, why do you still use the value `HDFS_ADDRESS` from cluster-object-model? You should know this value is parsed from `machine-list` in `cluster-configuration.yaml`

Reply:
And the PR about zk also confused me a lot.

Reply:
Hi @ydye ,
I'm refactoring step by step, and I don't expect a breaking change which will block anyone.
So I will decouple the cluster-object-module from the most edge side.